amr-wind/wind_energy/ABL.H

```diff
@@ -90,6 +91,7 @@ public:
 
     const ABLBoundaryPlane& bndry_plane() const { return *m_bndry_plane; }
     const ABLModulatedPowerLaw& abl_mpl() const { return *m_abl_mpl; }
+    const ABLCodedInlet& abl_coded_inlet() const { return *m_abl_coded_inlet; }
```

Comment:
instead of "CodedInlet" I would favor renaming to UDF or UserDefined

Reply:
"UDF" makes sense, but there are already UDFs in other parts of the code. I'll rename to UserDefined.